client/src/actions/authActions.js

```diff
@@ -42,6 +42,7 @@ export const loginUser = userData => dispatch =>
 			// Get user's habits
 			dispatch(getHabits());
 		})
+		.then(() => history.push('/dashboard'))
```

Comment:
Quick note: It would be better if we use `history.replace` here, to replace `/login` from history stack with `/dashboard`, but this should be handled with private route guards anyway, or maybe not 🤔

Reply:
I see what you mean with using `history.replace` (or `redirect`) in some cases but maybe push is better here. The user is on the login page, they log in, and then they are taken to the dashboard when successful. The login page still shows up in their browser history and that seems logical. If they are logged in and they try to navigate back to it, it would redirect, but if they log out and navigate back to it, it would display.

The withAuth component should stop them from accessing any private routes if they are not authenticated, no?

Reply:
I'll go ahead and merge now, this is something we can think about and change in the future 😀